From 3d405ac9a83666281e726275fe7f2962c56b7cd9 Mon Sep 17 00:00:00 2001
From: Zygmunt Krynicki <me@zygoon.pl>
Date: Wed, 2 Aug 2017 19:57:53 +0200
Subject: [PATCH] gitignore: ignore more build artefacts

Those files are made during the build but are reported by run-checks as
unknown junk. We can fix this by just marking them as ignored.

Signed-off-by: Zygmunt Krynicki <me@zygoon.pl>
---
 .gitignore | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.gitignore b/.gitignore
index 0afd0c6cb..b9f8c3fdb 100644
--- a/.gitignore
+++ b/.gitignore
@@ -19,6 +19,7 @@ po/snappy.pot
 cmd/decode-mount-opts/decode-mount-opts
 cmd/libsnap-confine-private/unit-tests
 cmd/snap-confine/snap-confine
+cmd/snap-confine/snap-confine-debug
 cmd/snap-confine/snap-confine.apparmor
 cmd/snap-confine/unit-tests
 cmd/snap-discard-ns/snap-discard-ns
@@ -32,6 +33,7 @@ cmd/*/*.[1-9]
 
 # auto-generated systemd units
 data/systemd/*.service
+data/info
 
 # test-driver
 *.log
-- 
2.13.3

